src/lifecycle.mjs

```diff
@@ -140,14 +140,14 @@ export async function init(worker) {
     const strategy = finder(lifeCycleType, message.name);
     const messages = await run(strategy, lifeCycleType, "init", message.args);
     messages.worker.forEach((message) => worker.postMessage(message));
-    messages.lifecycle.forEach((message) => lch.emit("message", [message]));
+    messages.lifecycle.forEach((message) => lch.emit("message", message));
   });
 
   lch.emit("message", {
     type: "extraction",
     version: "0.0.1",
     name: "web3subgraph",
-    args: null,
+    args: [null],
```

Comment:
Due to the destructuring of params the args is expected to be an array.

https://github.com/neume-network/strategies/blob/be9695224ef71e0a8048e498ef22350422a1510c/src/lifecycle.mjs#L101

Reply:
ah that's annoying, created an issue: https://github.com/neume-network/strategies/issues/57